static/js/components/forms/CheckoutForm.js

```diff
@@ -353,7 +353,7 @@ export class InnerCheckoutForm extends React.Component<InnerProps, InnerState> {
                       <span>{formatPrice(calculatePrice(item, coupon))}</span>
                     </div>
                     <div className="flex-row tax-row">
-                      <span>Tax:</span>
+                      <span>Tax ({basket.tax_info.tax_rate}%):</span>
```

Comment:
Could we change the floating points to 2?
<img width="394" alt="Screen Shot 2023-10-10 at 8 54 39 PM" src="https://github.com/mitodl/mitxpro/assets/52656433/3c8f044d-262d-41af-8fae-e420bd1ff6a7">